tests/acceptance/helpers/userSettings.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
     }
   },
   createdUsers: {},
+  createdGroups: {},
```

Comment:
Maybe it's better we just create `createdGroups` as an array.
Later if we need other attributes in a created group we can add it.